### ISSUE TITLE
[BUG] Impossible to select multiple tags

### ISSUE THREAD

**Issue by anhgelus** [anhgelus/blog-anhgelus#3]
## What happened?

When we create an article, we cannot select multiple tags.

## Expected behavior

Selecting multiple tags

## How to reproduce

1. Create an article
2. Try to select multiple tags

**Comment by kareninakvam** [comment]
Hello! Can I help u in this issue?

**Comment by anhgelus** [comment]
@kareninakvam yes, of course. The bug is in the html, not in the backend part.

And I'm pretty sure it is in this line but I don't really take the time to look up more information about it.
 https://github.com/anhgelus/blog-anhgelus/blob/253d3efa9b2081789526ecb2e29e90703c8cdcec/resources/views/admin/article/new.blade.php#L37
